### ISSUE TITLE
Update README translation tables with German file

### ISSUE THREAD

**Issue by erictleung** [freeCodeCamp/how-to-contribute-to-open-source#297]
There was a new language added to the list of READMEs. This language, German, needs to be added to the table.

In other words, this is the change that needs to be made to all `README-*.md` and `README.md` files.

```diff
<table>
    <tr>
        <!-- Do not translate this table -->
        <td><a href="./README.md"> English </a></td>
        <td><a href="./README-CN.md"> 中文 </a></td>
        <td><a href="./README-RU.md"> русский </a></td>
        <td><a href="./README-RO.md"> Românesc </a></td>
        <td><a href="./README-IT.md"> Italiano </a></td>
        <td><a href="./README-ES.md"> Español </a></td>
        <td><a href="./README-pt-BR.md"> Português (BR) </a></td>
+       <td><a href="./README-DE.md"> German </a></td>
    </tr>
</table>
```

**Comment by peterhuerlimann** [comment]
Hi @erictleung, 
I opened pull request #299 with the changes. 
As the other languages were localized in their own, I figured to add it in german, "Deutsch" instead of "German".

**Comment by erictleung** [comment]
Closed by #299

**Comment by erictleung** [comment]
@peterhuerlimann good call on localizing it in its own language :+1:
